packages/roosterjs-editor-api/lib/format/setFontWeight.ts

```diff
@@ -0,0 +1,24 @@
+import applyListItemStyleWrap from '../utils/applyListItemWrap';
+import { getComputedStyle } from 'roosterjs-editor-dom';
+import { IEditor } from 'roosterjs-editor-types';
+
+/**
+ * Set font weight at selection
+ * Only sets the value if the computed weight is different
+ * @param editor The editor instance
+ * @param fontWeight The fontWight string, should be a valid CSS font-weight style.
+ * Currently there's no validation to the string, if the passed string is invalid, it won't take affect
+ */
+export default function setFontWeight(editor: IEditor, fontWeight: string) {
+    applyListItemStyleWrap(
+        editor,
+        'font-weight',
+        element => {
+            let computedFontWeight = getComputedStyle(element, 'font-weight');
```

Comment:
Why not always set the font-weight? If we always set the font-wight, we will not need to call getComputedStyles and the function will be cheaper to call.

Reply:
True, this function is only called by clearFormat anyway